### PR TITLE
Content Add/View Implemented

### PR DESCRIPTION
**Pull request by XavierXinweiWang**
Nodes list does not persist for now.
Edit From the current master.
1-19 9:34am

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ var mouseDown = 0;
 var delay = 300;
 var mouseMoved = false;
 var singleClickTimer, duobleClickDragTimer = null;
+var mouseHoverNode = false;
+var hoveredNode = null;
+var mouseHoverEdge = false;
+var hoveredEdge = null;
```
Comment (IsaacFehr) [inline src/main.js]: instead of having two hovered flags, make one "hoveredObject" flag

File: src/main.js [hunk 2]
```diff
@@ -122,6 +128,79 @@ function mouseMoveListener(e) {
...
 }
 
+
+function mouseOverListener(e) {
+  let className = e.target.getAttribute("class").split(" ")[0];
+  
+  switch(className) {
+    case "node":
+      mouseHoverNode = true;
+      hoveredNode = e.target.getAttribute("id");
```
Comment (IsaacFehr) [inline src/main.js]: change to hoveredNodeId for descriptivity

File: src/main.js [hunk 3]
```diff
@@ -122,6 +128,79 @@ function mouseMoveListener(e) {
...
+      break;
+    default:
+      break;
+  }
+}
+
+function keyPressListener(e) {
+  var keyCode = e.which;
+    
+  if (keyCode == 97) {
```
Comment (IsaacFehr) [inline src/main.js]: Change to switch statement and add comment explaining keycode

**Comment by XavierXinweiWang** [comment]
I made some changes and refactoring all of my code for further review

**Pull request closed without merging**